test/integration/crud/crud.spec.test.js

```diff
@@ -425,6 +425,10 @@ describe('CRUD spec v1', function () {
   }
 });
 
+// The Node driver does not have a Collection.modifyCollection helper.
+const SKIPPED_TESTS = ['findOneAndUpdate document validation errInfo is accessible'];
```

Comment:
These tests are different from the collection management specs.  These tests use the `modifyCollection` operation to set up the collection for the test, but they're not testing the driver's `modifyCollection` helper.  

I think it might make sense to implement a unified test format helper for `modifyCollection` that uses `runCommand`to send a `collMod` to the server (i.e., a new operation in `operations.ts`).  Then we could unskip this test.  We'd still leave the `modifyCollection` tests alone though because we don't have a driver helper.  

what do you think? 

(also see https://github.com/mongodb/specifications/pull/1316/files#r994171436)

Reply:
Perhaps we could merge these changes and make a separate subtask or even ticket for adding in the modifyCollection changes?

Reply:
Sounds good to me - if we file a separate ticket, can we add a `TODO(NODE-XXXX)` comment here so we don't forget about it?